EIPS/eip-draft-muldiv.md

```diff
@@ -0,0 +1,147 @@
+---
+eip: nnn
```

Comment:
```suggestion
eip: 4998
```

Since we try to discourage sniping "fancy" EIP numbers, and this PR was opened immediately after a spam bot posted issue 4999, I'm reassigning to a less special number.

Reply:
@SamWilsn PR was merged as EIP-5000?

Reply:
Yep, that was @lightclient's decision.